brigadier/accounts/migrations/0002_create_public_group.py

```diff
@@ -15,7 +15,7 @@ def create_public_group(apps, schema_editor):
 
     Group = apps.get_model('auth', 'Group')
     Permission = apps.get_model('auth', 'Permission')
-    User = apps.get_model('auth', 'User')
+    User = apps.get_model('accounts', 'MyUser')
```

Comment:
remember you should never change migrations
when you want your db model to be change - create a new migration!

Reply:
As it already have done, I won't correct this mistake. But I get this rule for the future.